src/cljs/orcpub/dnd/e5/spell_subs.cljs

```diff
@@ -437,6 +437,12 @@
  (fn [plugins _]
    (apply concat (map (comp vals ::e5/invocations) plugins))))
 
+(reg-sub
+ ::classes5e/plugin-boons
+ :<- [::e5/plugin-vals]
+ (fn [plugins _]
+   (apply concat (map (comp vals ::e5/boons) plugins))))
```

Comment:
`comp` isn't used very commonly these days, because the listed fn order is reversed. And as someone else mentioned, `mapcat` is preferred over `(apply concat (map ...))` Consider `(mapcat #(-> % ::e5/boons vals) plugins)` instead.

Reply:
(Just recalled a slight addendum, `comp` is used heavily with transducers for double reversal, but we're not doing that here...)